packages/@azure/servicebus/data-plane/test/stress/fixedNumberOfClientsMultipleQueues.ts

```diff
@@ -0,0 +1,62 @@
+import { ServiceBusClient, SendableMessageInfo } from "../../lib";
+
+const connectionString = "";
+
+const testDurationInMilliseconds = 60000 * 5 * 12 * 24 * 7; // 1 week
+
+const numOfClients = 10;
+
+let msgId = 1;
+
+let snapshotIntervalID: any;
+
+let isJobDone = false;
+
+async function main(): Promise<void> {
+  snapshotIntervalID = setInterval(snapshot, 5000); // Every 5 seconds
+  setTimeout(() => {
+    isJobDone = true;
+  }, testDurationInMilliseconds);
+
+  sendReceiveMessages();
+}
+
+async function sendReceiveMessages(): Promise<void> {
```

Comment:
If we plan to mimic a real world scenario, then I would imagine `sendReceiveMessages()` to work on a single client. And then call it 10 times from the `main()` function. This would result in the send/receive on each client to happen independently. Currently each send and receive is happening sequentially.

Reply:
The idea here was to specifically use multiple clients.

We can sync up offline with @AlexGhiondea

Reply:
> The idea here was to specifically use multiple clients which will be created only once

I agree. That doesnt have to change. What I am suggesting is the below
- create the multiple clients once
- create 1 sender and 1 receiver for each of these clients once
- make sendReceiveMessages to take the sender and receiver as input parameters
- call sendReceiveMessages 10 times parallelly

Reply:
@ramya-rao-a that sounds good - it will allow for more things to happen in parallel